Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -249,22 +642,46 @@ The following rules need to be followed, in addition to the guidelines described
    Make sure sequences are tested for all modes of operation, and as applicable, for winter, shoulder
    and summer days.
 
+#. Perform code text and, if applicable, figure inspection to:
```

Comment:
Remove to be concise. It should be clear that code and figures need to be inspected.